.idea/codeStyles/Project.xml

```diff
@@ -37,15 +37,6 @@
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value />
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
this shouldn't have been deleted, I believe it gets contributed by the latest kotlin plugin for Android Studio

Reply:
The reason may probably be their old version of Android Studio(that's what I encountered)

Reply:
@Rajnish23 If you can resolve this then I think we can get this merged